server/svix-server/src/v1/endpoints/endpoint/crud.rs

```diff
@@ -21,29 +25,39 @@ use crate::{
     db::models::{application, endpoint, endpointmetadata, eventtype},
     error::{HttpError, Result, ValidationErrorItem},
     v1::utils::{
+        apply_pagination_reverse, iterator_from_before_or_after,
         patch::{patch_field_non_nullable, UnrequiredField, UnrequiredNullableField},
         ApplicationEndpointPath, EmptyResponse, ListResponse, ModelIn, ModelOut, Pagination,
-        PaginationLimit, ValidatedJson, ValidatedQuery,
+        PaginationLimit, ReversibleIterator, ValidatedJson, ValidatedQuery,
     },
     AppState,
 };
 use hack::EventTypeNameResult;
 
+#[derive(Deserialize, Validate, JsonSchema)]
+pub struct ListEndpointsQueryParameters {
```

Comment:
Why this? We don't want this.

Reply:
Only because other endpoints with pagination offered it too, but it is not necessary, dropping.

Reply:
We have it in the other ones because they are time-based, it's not the case for endpoints/apps (time based doesn't matter).